CHANGELOG.develop

```diff
@@ -555,6 +569,18 @@ Benner and Paweł Siudak):
     - ~SPC h P s~ to start the profiler
     - ~SPC h P w~ to write the profiler report to a file
   - New ~SPC x t e~ to transpose sexps (thanks to Guido Kraemer)
+  - Added multiple key bindings to copy a file path (thanks to Codruț Constantin Gușoi):
+    - ~SPC f y c~ to show and copy current file absolute path with line and column number in the minibuffer
+    - ~SPC f y d~ to show and copy current directory absolute path in the minibuffer
+    - ~SPC f y l~ to show and copy current file absolute path with line number in the minibuffer
+    - ~SPC f y y~ to show and copy current file absolute path in the minibuffer
+    - ~SPC f y C~ to show and copy current file path relaitve to the project root with line and column number
+    - ~SPC f y D~ to show and copy current directory path relaitve to the project root
+    - ~SPC f y L~ to show and copy current file path relaitve to the project root with line number
+    - ~SPC f y Y~ to show and copy current file path relaitve to the project root
```

Comment:
The above four lines all have a typo: "relaitve" (the typo was present in the documentation too but has been fixed).